tests/modules/media_avportal_responsive_test/media_avportal_responsive_test.info.yml

```diff
@@ -0,0 +1,13 @@
+name: Media AV Portal Responsive Test
+description: Test module for the Responsive image style
+type: module
+core: 8.x
+package: Testing
+
+dependencies:
+  - drupal:image
+  - drupal:responsive_image
+
+config_devel:
```

Comment:
Do we need theoretically config_devel usage for test type modules?

Reply:
Agree with this, not sure of its usefulness on a test module.

Reply:
isnt config devel thing there to help exporting the config to the files? So what diff does it make that it's test or not?

Reply:
But as I understand usage of test module have short life-time and during usage of this module we shouldn't have cases when we need export or import config. Config will be applied only on install and after finishing execution of sandbox will be destroyed.